frontend/src/components/ReviewForm/InputTextBox.jsx

```diff
@@ -0,0 +1,19 @@
+const InputTextBox = ({ labelText, placeHolderText, minWidth, val, setVal }) => {
+  const handleChange = (e) => {
+    setVal(e.target.value);
+  };
+
+  return(
+    <div className="text-start gap-y-1 w-auto">
+      <p>{ labelText }</p>
+      <input
+        type="text"
+        placeholder={ placeHolderText }
+        className={`bg-transparent border border-white rounded-sm pl-3 h-9 ${minWidth}`}
+        onChange={ handleChange }
```

Comment:
I'm wondering, will the code become more complicated than setting a value?  If not maybe something like:

```suggestion
onChange={(e)=> setVal(e.target.value)}
```

Reply:
done